.github/workflows/pr.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '15.x'
+    
+    - name: Install dependencies
+      run: npm ci
```

Comment:
Good thinking in using `ci` here!

Reply:
I was reading about it and apparently is faster and better than `i` :)